Sources/Routes/ImageJPEG.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+import Request
+
+public class ImageJPEG: Route {
+    
+    public init() {}
+
+    public func handleRoute(request: Request) -> RouteData {
+        let body = "" 
+        let image = getImage(request: request) 
+        let responseLineData = packResponseLine(request: request) 
+        let headersData = packResponseHeaders(body: body, request: request)
+        return RouteData(responseLine: responseLineData, headers: headersData, body: body, image: image)
+    }
+    
+    private func getImage(request: Request) -> Data {
+        let filePath = NSURL.fileURL(withPathComponents: ["\(request.directory)/image.jpeg"])
+        var imageData = Data()
+        do {
+            imageData = try Data(contentsOf: filePath!)
+        } catch {
+            print("Cannot convert .jpeg image.")
+        }
+        return imageData
```

Comment:
So, if the image can't be read for any reason, the user just gets an empty response back? Or, do they get a 200 but no data?